test/notification.js

```diff
@@ -530,35 +534,37 @@ exports.testGlobalOptOut = function(test)
 exports.testMessageWithoutLocalization = function(test)
 {
   let notification = {message: "non-localized"};
-  let texts = Notification.getLocalizedTexts(notification, "en-US");
+  let texts = Notification.getLocalizedTexts(notification);
   test.equal(texts.message, "non-localized");
   test.done();
 };
 
 exports.testLanguageOnly = function(test)
 {
-  let notification = {message: {fr: "fr"}};
-  let texts = Notification.getLocalizedTexts(notification, "fr");
-  test.equal(texts.message, "fr");
-  texts = Notification.getLocalizedTexts(notification, "fr-CA");
-  test.equal(texts.message, "fr");
+  let notification = {message: {en: "en"}};
```

Comment:
Please can you change this test back to how it was, except now assigning the locales to `Utils.appLocale` instead of passing them as a parameter to `Notification.getLocalizedTexts`?